battery.py

```diff
@@ -0,0 +1,17 @@
+
+
+class Battery:
+    
+    def __init__(self, max_capacity, current_capacity = -1):
+        if(current_capacity == -1):
```

Comment:
not a functional thing, but I would usually put the default value here to `None` instead of -1